src/legacy/core_plugins/timelion/public/vis/index.ts

```diff
@@ -17,29 +17,21 @@
  * under the License.
  */
 
-import { VisFactoryProvider } from 'ui/vis/vis_factory';
+// @ts-ignore
```

Comment:
Are you sure we need // @ts-ignore for '@kbn/i18n' ?

Reply:
Removed